documentation/leo/03_language.md

```diff
@@ -412,106 +412,48 @@ program test.aleo {
 ```
 
 #### Mapping
-
 The mapping struct allows the programmer to apply updates to a program mapping data structure by calling one of the
 following functions.
 
 #### get
-
 A get command, e.g. `current_value = Mapping::get(counter, addr);`
 Gets the value stored at `addr` in `counter` and stores the result in `current_value`
 If the value at `addr` does not exist, then the program will fail to execute.
 
 #### get_or_init
-
 A get command that initializes the mapping in case of failure, e.g.
 `let current_value: u64 = Mapping::get_or_init(counter, addr, 0u64);`
 Gets the value stored at `addr` in `counter` and stores the result in `current_value`.
 If the key is not present, `0u64` is stored in `counter` and stored in `current_value`.
 
 #### set
-
 A set command, e.g. `Mapping::set(counter, addr, current_value + 1u64);`
 Sets the `addr` entry as `current_value + 1u64` in `counter`.
 
-## Iteration
-
-In the Leo programming language, iteration is supported through the `for` loop construct. This section provides an
-overview of how
-iteration works in Leo and highlights its limitations.
-
-### `for` Loop Syntax
+## For Loops
 
-The syntax for a `for` loop in Leo is as follows:
+For Loops are declared as `for <loop variable> in <lower bound>..<upper bound>`.  Currently, the
+loop variable supports unsigned integer types u8, u16, u32, or u64. The lower bound must be
+less than the upper bound.
 
-```
+```leo
 for <loop variable> in <lower bound>..<upper bound> {
     // Loop body
 }
 ```
 
-The `for` loop starts by declaring a loop variable that will take on different values in each iteration. The loop
-variable currently supports the `u8`, `u16`, or `u32` data types. It is then
-followed by the `in` keyword and a range specified by the lower and upper bounds.
+### Nested Loops
 
-### Example:
+You can nest loops within loops. For example:
```

Comment:
I don't think you need to provide an example here. Just say nested loops are supported.